packages/api-server/src/lib/data-extractors.ts

```diff
@@ -19,6 +34,14 @@ export async function getWritable(object: any, req: IncomingMessage, res: Server
     return object;
 }
 
+/**
+ * Identifies type of given object. If object is a function, it will be called.
+ *
+ * @param req Request object.
+ * @param res Response object.
+ * @param stream Object to identify.
+ * @returns Readable if object is a stream or results of function call if object is a function.
```

Comment:
Looking at the description and the code I'm not sure :thinking: If `stream` is a `Promise` or a `function` it gets resolved/called but its result is passed to `getStream` again - so I guess the purpose here is to get `stream` instance at the end or throw error? WDYT?